smart-contracts/src/util.rs

```diff
@@ -1,7 +1,10 @@
 use crate::error::ContractError;
 
 pub fn validate_color(color: &String) -> Result<String, ContractError> {
-    if color.starts_with('#') && color.len() == 7 && color.chars().skip(1).all(|c| c.is_digit(16)) {
+    if color.starts_with('#')
+        && color.len() == 7
+        && color.chars().skip(1).all(|c| c.is_ascii_hexdigit())
```

Comment:
maybe we can write a small test for this function?

Reply:
done